CHANGELOG.md

```diff
@@ -350,11 +350,11 @@ class Animal extends Organism {
 **Major changes**:
 
 * Builders now use `built_value`, and have a more consistent, friendly API.
-* Builders are now consistent - they don't any work until code is emitted.
+* Builders are now consistent - they don't work until code is emitted.
```

Comment:
This should probably read `they don't perform work until...` or `they don't do any work until...`.

Reply:
Done